moped-editor/src/views/projects/projectView/ProjectComponentsMap.js

```diff
@@ -274,32 +253,35 @@ const ProjectComponentsMap = ({
     }
   );
 
+  const shouldShowDrawTools = !isSignalComponent && drawLines !== null;
   /*
    * {boolean} isDrawing - Are draw tools enabled or disabled
-   * {function} setIsDrawing - Toggle isdrawing
-   * {function} renderMapDrawTools - Function that returns JSX for the draw tools in the map
    * {function} saveDrawnPoints - Function that saves features drawn in the UI
    */
-  // const { isDrawing, setIsDrawing, renderMapDrawTools, saveDrawnPoints } =
-  //   useMapDrawTools(
-  //     featureCollection,
-  //     setFeatureCollection,
-  //     projectId,
-  //     refetchProjectDetails,
-  //     viewport.zoom,
-  //     saveActionDispatch
-  //   );
+  const { isDrawing, saveDrawnPoints, renderMapDrawTools } = useMapDrawTools(
+    featureCollection,
+    setFeatureCollection,
+    saveActionDispatch,
+    drawLines
+  );
+
+  const [cursor, setCursor] = useState("auto");
+  const onMouseEnter = useCallback(() => setCursor("pointer"), []);
+  const onMouseLeave = useCallback(() => setCursor("auto"), []);
+
+  // Update cursor if we are drawing
+  useEffect(() => {
+    if (isDrawing) {
+      setCursor(() => "crosshair");
+    }
+  }, [isDrawing]);
 
   /**
    * Adds or removes an interactive map feature from the project's feature collection and selected IDs array
    * @param {Object} e - Event object for click
    */
   const handleLayerClick = (e) => {
     const layerName = getLayerSource(e);
-    // If a user clicks a drawn point in the map, open draw UI
-    // if (drawnLayerNames.includes(layerName)) {
-    //   setIsDrawing(true);
-    // }
```

Comment:
This "click on a drawn point to open the draw tools" functionality is disabled elsewhere in the code so I removed this since this isn't how the draw UI works before or after this code update.